hummingbot/market/bamboo_relay/bamboo_relay_market.pyx

```diff
@@ -250,39 +282,54 @@ cdef class BambooRelayMarket(MarketBase):
     MARKET_ORDER_EXPIRED_EVENT_TAG = MarketEvent.OrderExpired.value
 
     API_CALL_TIMEOUT = 10.0
+    CANCEL_EXPIRY_TIME = 60.0
     ORDER_EXPIRY_TIME = 60.0 * 15
+    PRE_EMPTIVE_SOFT_CANCEL_TIME = 30.0
+    ORDER_CREATION_BACKOFF_TIME = 3
     UPDATE_RULES_INTERVAL = 60.0
     UPDATE_OPEN_LIMIT_ORDERS_INTERVAL = 10.0
     UPDATE_MARKET_ORDERS_INTERVAL = 10.0
 
     @classmethod
     def logger(cls) -> HummingbotLogger:
-        global rrm_logger
-        if rrm_logger is None:
-            rrm_logger = logging.getLogger(__name__)
-        return rrm_logger
+        global brm_logger
+        if brm_logger is None:
+            brm_logger = logging.getLogger(__name__)
+        return brm_logger
 
     def __init__(self,
                  wallet: Web3Wallet,
                  ethereum_rpc_url: str,
+                 chain: EthereumChain = EthereumChain.MAIN_NET,
```

Comment:
the initialized wallet instance knows which chain its on. check line 147 on web3_wallet_backend.py.

Reply:
I exposed it on Web3Wallet as a property, given that makes the most sense?